packages/mobile/src/app/App.tsx

```diff
@@ -111,7 +112,9 @@ export class App extends React.Component<Props> {
             <PersistGate loading={<AppLoading />} persistor={persistor}>
               <StatusBar backgroundColor="transparent" barStyle="dark-content" />
               <ErrorBoundary>
-                <NavigatorWrapper />
+                <I18nGate fallback={<AppLoading />}>
```

Comment:
Why not directly as a child of `PersistGate`?

Reply:
I tried to block as little as possible and thought maybe a good idea for the I18nGate to have the ErrorBoundary to fall back on...although now I can see that ErrorBoundary also depends on i18n :/ so yeah good shout, i will move this below PersistGate